org.uqbar.project.wollok.lib/src/wollok/lang/Range.xtend

```diff
@@ -39,8 +40,12 @@ class Range extends AbstractJavaWrapper<IntegerRange> {
 	
 	def Integer validate(Object value) {
 		try {
-			return value.wollokToJava(Integer) as Integer
-		} catch (Throwable e) {
+			val castedValue = value.wollokToJava(BigDecimal) as BigDecimal
```

Comment:
I think this cast could be avoided if we add a generic return type to wollokTojava:
`def <T> T wollokToJava(Class<? extends T> type)`, isn't it?